src/app/common/facetData.js

```diff
@@ -37,6 +37,67 @@ const facetData = [
     label: 'Facet not shown',
     name: 'facet_not',
   },
+  {
+    label: 'Long facet shown',
+    name: 'long_facet',
+    values: [
```

Comment:
There are duplicate `value_1` attributes for `value` in this array and because the `id` for the elements sent to the `ExpandableList` are made with `${facet.name}_${value.value}` this is producing duplicate key errors in the console.

Reply:
Well spotted, just updated it.